python/lsst/analysis/drp/addQAColumns.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import lsst.pipe.base as pipeBase
+
+from .qaUtils import (addMagnitudes, addColors, stellarLocusFits, addUseForQAFlag, addUseForStatsColumn,
+                      addSNColumn, addDeconvMoments)
+
+
+class AddQAColumnsTaskConnections(pipeBase.PipelineTaskConnections, dimensions=("tract", "skymap")):
+
+    cat = pipeBase.connectionTypes.Input(doc="The tract wide catalog to make plots from.",
+                                         storageClass="DataFrame",
+                                         name="objectTable_tract",
+                                         dimensions=("tract", "skymap"))
+
+    qaCat = pipeBase.connectionTypes.Output(doc="The input catalog with additional columns for QA added",
+                                            storageClass="DataFrame",
+                                            name="qaTable_tract",
+                                            dimensions=("tract", "skymap"))
+
+
+class AddQAColumnsTaskConfig(pipeBase.PipelineTaskConfig, pipelineConnections=AddQAColumnsTaskConnections):
+
+    pass
+
+
+class AddQAColumnsTask(pipeBase.PipelineTask):
```

Comment:
I'd like this to be named `AppendQAColumnsTask` (as well as `add` --> `append` in many other places.  "add" suggests to me a mathematical operation, whereas appending columns is what this is doing.